Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -3262,6 +3298,62 @@ public AndConstraint<TAssertions> StartWith(T element, string because = "", para
         return StartWith(new[] { element }, ObjectExtensions.GetComparer<T>(), because, becauseArgs);
     }
 
+    internal void AssertContainment(IEnumerable<T> expected, string containmentType, string because = "", params object[] becauseArgs)
```

Comment:
🙃 We prefer to keep private methods just below the last usage.
🔧 Make it `private`